packages/protobuf/src/google/varint.ts

```diff
@@ -161,19 +161,47 @@ export function int64fromString(dec: string): [boolean, number, number] {
   add1e6digit(-18, -12);
   add1e6digit(-12, -6);
   add1e6digit(-6);
-  return [minus, lowBits, highBits];
+  return minus ? negate(lowBits, highBits) : newBits(lowBits, highBits);
 }
 
 /**
- * Format 64 bit integer value (as two JS numbers) to decimal string.
+ * Losslessly converts a 64-bit signed integer in 32:32 split representation
+ * into a decimal string.
  *
- * Copyright 2008 Google Inc.
+ * Copyright 2008 Google Inc.  All rights reserved.
+ *
+ * See https://github.com/protocolbuffers/protobuf-javascript/blob/a428c58273abad07c66071d9753bc4d1289de426/experimental/runtime/int64.js#L10
  */
-export function int64toString(bitsLow: number, bitsHigh: number): string {
+export function int64ToString(lo: number, hi: number): string {
```

Comment:
This could have been a much smaller patch, but it seemed more reasonable to keep this in sync.